CommonClient.py

```diff
@@ -278,6 +279,17 @@ def __init__(self, server_address: typing.Optional[str], password: typing.Option
         # execution
         self.keep_alive_task = asyncio.create_task(keep_alive(self), name="Bouncy")
 
+        if gui_enabled:
```

Comment:
since this is in the constructor, it's expected to be run for all subclasses
```suggestion
        if gui_enabled and self.ui_manager is None:
```